charts/splunk-connect-for-snmp/templates/traps/deployment.yaml

```diff
@@ -59,6 +59,8 @@ spec:
               value: "http://{{ printf "%s-%s" .Release.Name "mibserver" }}/standard.txt"
             - name: LOG_LEVEL
               value: {{ .Values.traps.logLevel | default "INFO" }}
+            - name: PYSNMP_DEBUG
```

Comment:
traps haven't `_helper.tpl` like have workers that can reuse it, by this reason for traps it on deployment